package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "the-restaurant-page",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "dev": "webpack --mode development",
+        "build": "webpack --mode production",
+        "start": "webpack-dev-server --mode development --open"
```

Comment:
- [ ] If the webpack dev server option is being used please include the `webpack-dev-server` package in the project.